src/lib/components/ResultsMultiLayout/ResultsMultiLayout.js

```diff
@@ -33,20 +34,20 @@ ResultsMultiLayout.propTypes = {
 ResultsMultiLayout.defaultProps = {
   currentLayout: null,
   overridableId: "",
+  onResultsRendered: () => {},
 };
 
-const Element = ({ layout, overridableId }) => {
+const Element = ({ layout, overridableId, onResultsRendered }) => {
```

Comment:
Sorry, last thing: the Overridable cmp *always* require that you pass *all* params. `onResultsRendered` needs to be passed `onResultsRendered={onResultsRendered}`.
Double-check the other cmps above too.